composer.json

```diff
@@ -12,8 +12,8 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/serializer": "~2.7",
-        "symfony/options-resolver": "~2.7"
+        "symfony/serializer": ">=2.7,<=3.0",
+        "symfony/options-resolver": ">=2.7,<=3.0"
```

Comment:
The right way would be: `"symfony/options-resolver": ">=2.7"`

Same for serializer.

I don't see any reason lock to the specific `3.0` version. When Symfony `3.1` will be released we will need another PR to fix this...